### PR TITLE
Mcol 4560

### PR DESCRIPTION
**Pull request by davidhall586**

### CODE REVIEW

File: dbcon/joblist/primitivestep.h [hunk 1]
```diff
@@ -224,8 +182,10 @@ class pColStep : public JobStep, public PrimitiveMsg
    *
```
Comment (tntnatbry) [inline dbcon/joblist/primitivestep.h:177]: `setOutputType()` function is not used and should be deleted.

File: oam/etc/Columnstore.xml [hunk 2]
```diff
@@ -96,203 +40,38 @@
...
-		<ModuleMemMinorThreshold1>0</ModuleMemMinorThreshold1>
-		<ModuleSwapCriticalThreshold1>90</ModuleSwapCriticalThreshold1>
-		<ModuleSwapMajorThreshold1>80</ModuleSwapMajorThreshold1>
-		<ModuleSwapMinorThreshold1>70</ModuleSwapMinorThreshold1>
-		<ModuleDiskMonitorFileSystem1-1>/</ModuleDiskMonitorFileSystem1-1>
-		<ModuleDBRootCount1-1>unassigned</ModuleDBRootCount1-1>
-		<ModuleDBRootID1-1-1>unassigned</ModuleDBRootID1-1-1>
 		<ModuleType2>um</ModuleType2>
 		<ModuleDesc2>User Module</ModuleDesc2>
 		<RunType2>SIMPLEX</RunType2>
```
Comment (tntnatbry) [inline oam/etc/Columnstore.xml]: `RunType2` should be removed.

**Review by tntnatbry — CHANGES_REQUESTED** [review]

File: dbcon/joblist/primitivestep.h (hunk 1 above)
Comment (davidhall586) [reply dbcon/joblist/primitivestep.h:177]: Removed

File: oam/etc/Columnstore.xml (hunk 2 above)
Comment (davidhall586) [reply oam/etc/Columnstore.xml]: removed

**Review by tntnatbry — APPROVED** [review]

**Pull request merged**